chihuahua/assetlist.json

```diff
@@ -104,6 +104,29 @@
       "images": [
         {
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/woof.png"
+        },
+        {
+      "description": "Laika the Space Dog",
+      "denom_units": [
+        {
+          "denom": "factory/chihuahua19z6pmhh34vu08d4d4qt2sen8dg4ja6ulyg209j/LAIKA",
+          "exponent": 0
+        },
+        {
+          "denom": "LAIKA",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua19z6pmhh34vu08d4d4qt2sen8dg4ja6ulyg209j/LAIKA",
+      "name": "LAIKA",
+      "display": "LAIKA",
+      "symbol": "LAIKA",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/laika.svg"
```

Comment:
Incorrectly have the image url using .svg instead of .png (should also point to neutron's laika image)